.github/workflows/HardhatTests.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, and run hardhat tests on Arrow contracts
```

Comment:
I think the multiple file extensions are confusing. Maybe worth just calling it `HardhatTests.yml`.